packages/web-react/src/components/ModalComposed/ModalDialog.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useStyleProps } from '../../hooks';
+import { ModalDialogProps } from '../../types/modalComposed';
+import { useModalStyleProps } from './useModalStyleProps';
+
+const ModalDialog = <E extends ElementType = 'article'>(props: ModalDialogProps<E>) => {
+  const { elementType, children, ...restProps } = props;
+
+  const { classProps } = useModalStyleProps({});
+  const { styleProps, props: otherProps } = useStyleProps(restProps);
+
+  const Element = elementType || 'article';
```

Comment:
Please check the variable name in every Modal component. We use `ElementTag` in every component written.
```suggestion
  const ElementTag = elementType || 'article';
```